docs/CLI.md

```diff
@@ -0,0 +1,87 @@
+# SkyWalking Python Agent Command-Line Interface(CLI)
+
+You would at least need to add the following lines to your applications in previous releases to get the agent attached and running.  
```

Comment:
What about putting the condition at the beginning? In case the users just read a half of the sentence 🤣 and let's be clear about since which version we support this

```suggestion
In earlier releases than 0.7.0, you would at least need to add the following lines to your applications to get the agent attached and running.  
```